app/src/main/java/org/sil/storyproducer/controller/MainActivity.kt

```diff
@@ -139,7 +168,8 @@ class MainActivity : BaseActivity(), Serializable {
             }
             R.id.nav_demo -> {
                 Workspace.addDemoToWorkspace(this)
-                storyList.notifyDataSetChanged()
+//                storyList.notifyDataSetChanged()
+                // FILTER TODO: notify the current story lists, this needs to be added back in
```

Comment:
@bryanwussow Yes, this was given to Claire for completion. This PR is not quite completed in the area of adding the demo into the Story List, and all of it needs testing before committing into develop. I need to go through this PR and refine improvements that have been made since my initial redesign. Not all the "FILTER TODO:" instructions I left have been completed.

Reply:
@bryanwussow This is fixed in the version I just pushed.